### PR TITLE
feat(statics): create statics for avaxp

### PR DESCRIPTION
**Pull request by sskim16**
add statics for avaxp with generic utxo for future utxo coins that are
not a btc derivative

STLX-15947

### CODE REVIEW

File: modules/bitgo/test/v2/unit/keychains.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ describe('V2 Keychains', function () {
     const cryptoSecpCoins = coins.filter(n => n.primaryKeyCurve === KeyCurve.Secp256k1
       && n.kind === CoinKind.CRYPTO
       && n.asset !== UnderlyingAsset.USD
+      && n.asset !== UnderlyingAsset.AVAXP
```
Comment (sskim16) [inline modules/bitgo/test/v2/unit/keychains.ts:57]: skip underlying assets test temporarily, enable once we have support in SDK

File: modules/statics/src/avax-p.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+   * Additional fields for utxo coins
+   */
+  public readonly network: AvalancheNetwork;
+
+  constructor(options: UtxoConstructorOptions) {
+    super({
+      ...options,
+      kind: CoinKind.CRYPTO,
+      isToken: false,
+      decimalPlaces: 8,
```
Comment (jrafael-simtlix) [inline modules/statics/src/avax-p.ts:32]: Avax c has 9 decimal, becouse it's half of avax c

Comment (sskim16) [reply modules/statics/src/avax-p.ts]: fixed

File: modules/statics/src/networks.ts [hunk 3]
```diff
@@ -100,6 +104,22 @@ class AvalancheCTestnet extends Testnet implements AccountNetwork {
...
+  explorerUrl = 'https://pchain.explorer.avax.network/tx/';
+  accountExplorerUrl = 'https://pchain.explorer.avax.network/address/';
+  blockchainID = '11111111111111111111111111111111LpoYY';
+}
+
+class AvalanchePTestnet extends Testnet implements AvalancheNetwork {
+  name = 'AvalanchePTestnet';
+  family = CoinFamily.AVAXP;
+  explorerUrl = 'https://pchain.explorer.avax-test.network/tx/';
+  accountExplorerUrl = 'https://pchain.explorer.avax-test.network/address/';
```
Comment (jrafael-simtlix) [inline modules/statics/src/networks.ts:119]:
```suggestion
  explorerUrl = 'https://explorer-xp.avax-test.network/tx/';
  accountExplorerUrl = 'https://explorer-xp.avax-test.network/address/';
```

File: modules/statics/src/networks.ts [hunk 4]
```diff
@@ -100,6 +104,22 @@ class AvalancheCTestnet extends Testnet implements AccountNetwork {
   chainId = 43113;
 }
 
+class AvalancheP extends Mainnet implements AvalancheNetwork {
+  name = 'AvalancheP';
+  family = CoinFamily.AVAXP;
+  explorerUrl = 'https://pchain.explorer.avax.network/tx/';
+  accountExplorerUrl = 'https://pchain.explorer.avax.network/address/';
```
Comment (jrafael-simtlix) [inline modules/statics/src/networks.ts:111]:
```suggestion
  explorerUrl = 'https://explorer-xp.avax.network/tx/';
  accountExplorerUrl = 'https://explorer-xp.avax.network/address/';
```

File: modules/statics/src/networks.ts (hunk 3 above)
Comment (sskim16) [reply modules/statics/src/networks.ts]: fixed

File: modules/statics/src/networks.ts (hunk 4 above)
Comment (sskim16) [reply modules/statics/src/networks.ts]: fixed

**Review by jrafael-simtlix — APPROVED** [review]

File: modules/statics/src/avax-p.ts [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  primaryKeyCurve: KeyCurve;
+}
+
+export class UtxoCoin extends BaseCoin {
+  public static readonly DEFAULT_FEATURES = [
+    CoinFeature.UNSPENT_MODEL,
+    CoinFeature.CHILD_PAYS_FOR_PARENT,
+    CoinFeature.CUSTODY,
+  ];
+
```
Comment (shiksb) [inline modules/statics/src/avax-p.ts:21]: get rid of this

File: modules/statics/src/avax-p.ts [hunk 6]
```diff
@@ -0,0 +1,82 @@
+import { BaseCoin, CoinFeature, CoinKind, KeyCurve, UnderlyingAsset } from './base';
+import { AvalancheNetwork } from './networks';
+
+export interface UtxoConstructorOptions {
```
Comment (shiksb) [inline modules/statics/src/avax-p.ts:4]: rename to AvaxPConstructorOptions

File: modules/statics/src/avax-p.ts [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  name: string;
+  network: AvalancheNetwork;
+  features: CoinFeature[];
+  asset: UnderlyingAsset;
+  prefix?: string;
+  suffix?: string;
+  primaryKeyCurve: KeyCurve;
+}
+
+export class UtxoCoin extends BaseCoin {
```
Comment (shiksb) [inline modules/statics/src/avax-p.ts:15]: AvaxPCoin

File: modules/statics/src/networks.ts [hunk 8]
```diff
@@ -18,6 +18,10 @@ export interface UtxoNetwork extends BaseNetwork {
   utxolibName: string;
 }
 
+export interface AvalancheNetwork extends BaseNetwork {
+  blockchainID: string;
+}
```
[the remark this answers is not in the document]
Comment (sskim16) [reply modules/statics/src/networks.ts:42]: done

File: modules/statics/src/coins.ts [hunk 9]
```diff
@@ -29,6 +29,7 @@ import { CoinMap } from './map';
 import { Networks } from './networks';
 import { ofc, ofcerc20, tofc, tofcerc20 } from './ofc';
 import { utxo } from './utxo';
+import { avaxP } from './avax-p';
```
Comment (shiksb) [inline modules/statics/src/coins.ts:32]:
```suggestion
import { avaxp } from './avaxp';
```

**Review by shiksb — APPROVED** [review]
Nice work!

**Pull request merged**